attestation-service/attestation-service/src/lib.rs

```diff
@@ -204,6 +205,7 @@ impl AttestationService {
             .collect();
 
         let token_claims = json!({
+            "tee": to_variant_name(&tee)?,
```

Comment:
nit: https://github.com/confidential-containers/kbs/blob/main/attestation-service/docs/example.token.json should also be updated.